go.mod

```diff
@@ -6,7 +6,7 @@ require (
 	github.com/Masterminds/goutils v1.1.0 // indirect
 	github.com/Masterminds/semver v1.4.2 // indirect
 	github.com/Masterminds/sprig v2.20.0+incompatible // indirect
-	github.com/deislabs/oras v0.5.0 // indirect
+	github.com/deislabs/oras v0.7.0+incompatible //indirect
```

Comment:
github.com/deislabs/oras v0.7.0+incompatible is bad module this is why it fails with gocenter.io which checks and validates all modules.
Please change to github.com/deislabs/oras v0.6.0

Reply:
There is also an errors with that proxy like ```cannot find module providing package k8s.io/cloud-provider/features``` & ```go: k8s.io/kubernetes@v1.15.0: unexpected status (https://gocenter.io/k8s.io/kubernetes/@v/v1.15.0.info): 404 Not Found```. 

v.0.7.0 is used by Helm 3 as far as I see in their Gopkg.

Reply:
`Helm 3-beta.x` needs `oras 0.7.0` to build. Build error otherwise:
```console
# helm.sh/helm/internal/experimental/registry
../../go/pkg/mod/helm.sh/helm@v3.0.0-beta.2+incompatible/internal/experimental/registry/client.go:96:27: too many arguments in call to c.authorizer.Client.Login
	have ("context".Context, string, string, string, bool)
	want ("context".Context, string, string, string)
```